mzLib/MassSpectrometry/MsDataScan.cs

```diff
@@ -270,6 +270,11 @@ public void SetOneBasedScanNumber(int value)
             this.OneBasedScanNumber = value;
         }
 
+        public void SetMsnOrder(int value)
```

Comment:
What's the reason for this change? I think it makes sense that the order should only be set on initialization